internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -28,6 +30,7 @@ var (
 	errNoSourceVolume           = errors.New("`source_volume` cannot be empty")
 	errEmptyEFSConfig           = errors.New("bad EFS configuration: `efs` cannot be empty")
 	errMissingPublishTopicField = errors.New("field `publish.topics[].name` cannot be empty")
+	errDeadLetterQueueTries     = errors.New("DeadLetter `tries` field cannot exceed 1000")
```

Comment:
Can we constantize this max value, along with the other bounds, and create error messages that include those constants using `fmt.Errorf()`?